VSConfigFinder/IFileSystem.cs

```diff
@@ -13,13 +13,13 @@ public interface IFileSystem
         /// <summary>
         /// Get files and directories within directory.
         /// </summary>
-        /// <param name="path">directory path</param>
+        /// <param name="path">directory paths to search for.</param>
         /// <param name="pattern">pattern to match for file names. To match anything and everything, specify '*'</param>
         /// <param name="recursive">Optional: recursively search sub directories</param>
         /// <returns>array of files within the directory</returns>
         /// <exception cref="ArgumentException"><paramref name="path"/> is empty.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="path"/> is null.</exception>
-        public string[] GetFileSystemEntries(string path, string pattern, bool recursive = false);
+        public IEnumerable<string> GetFileSystemEntries(IEnumerable<string> paths, string pattern, bool recursive = false);
```

Comment:
nit: I think we should keep the input a single path, and add an extension or a method overload that takes in an IEnumerable<string>. Keep this a more general-purpose interface

Reply:
agreed! added an extension method